### PR TITLE
Adding a bonus challenge

### PR DESCRIPTION
**Pull request by Xiaohua-Eric-Xu**

### CODE REVIEW

File: geodesy/README.md [hunk 1]
```diff
@@ -148,9 +148,11 @@ Last, we add some details like beach ball, epicenters, etc to the map.
 
 <img src="demo7.jpg" width="40%">
 
-### Bonus challenge
+### Bonus challenges
 
-Figure out what's inside the final map and complete the rest of the plot. 
+Figure out what's inside the final map and complete the rest of the plot. Focal mechanisms and epicenters could be find here: 7.1 https://earthquake.usgs.gov/earthquakes/eventpage/ci38457511/executive and 6.4 https://earthquake.usgs.gov/earthquakes/eventpage/ci38443183/executive
```
Comment (PaulWessel) [inline geodesy/README.md]: could be find here: 7.1 https://earthquake.usgs.gov/earthquakes/eventpage/ci38457511/executive and 6.4

Comment (PaulWessel) [reply geodesy/README.md]: Probably should say" can be found here"

**Review by PaulWessel — APPROVED** [review]

**Comment by Xiaohua-Eric-Xu** [comment]
I made the changes but is it reflected here?

**Comment by PaulWessel** [comment]
No, I guess things are different when you are not one of the owners of gmt github.  I can fix those two words after merging this one.

**Comment by PaulWessel** [comment]
A pushed a few spelling fixes directly to main, so if you git pull you should get the latest.

**Pull request merged**